third_party/mediapipe_calculators/BUILD

```diff
@@ -105,13 +102,20 @@ cc_library(
         #VISIBILITY_PRIVATE_THUS_SINK_BELOW"@mediapipe//mediapipe/calculators/internal:callback_packet_calculator",
         "@mediapipe//mediapipe/framework/tool:sink",
         "@mediapipe//mediapipe/calculators/video:flow_to_image_calculator",
-        "@mediapipe//mediapipe/calculators/video:tvl1_optical_flow_calculator",
         "@mediapipe//mediapipe/calculators/video:motion_analysis_calculator",
         "@mediapipe//mediapipe/calculators/video:flow_packager_calculator",
         "@mediapipe//mediapipe/calculators/video:box_tracker_calculator",
         "@mediapipe//mediapipe/calculators/video:box_detector_calculator",
         "@mediapipe//mediapipe/calculators/video:tracked_detection_manager_calculator",
         "@mediapipe//mediapipe/calculators/video:video_pre_stream_calculator",
-    ],
+    ]  + select({
+        "//conditions:default": [
+            "@mediapipe//mediapipe/calculators/core:packet_cloner_calculator", # TODO windows: stdc++20 required
```

Comment:
You probably need to adjust whitelist of calculators and adjust for Win/Lin

Reply:
This will be done when enabling tests.